README.md

```diff
@@ -43,6 +43,9 @@ Create a For Loop that outputs the following string for each piece of fruit in t
 
 ```js
 // write code here
+for (var i = 0; i <= foods[0].length; i++) {
```

Comment:
This for loop should actually be : 
```
for (var i = 0; i < foods[0].length; i++) {
   console.log("I want to eat a" + foods[0][i]);
};
```
In this case, if we do  `i <= foods[0].length` its going to console.log "I want to eat aundefined", because it is going it start at index[0] and iterate 4 times instead of 3